activities.html

```diff
@@ -64,12 +28,11 @@
       <nav>
           <ul id="menu">
               <li><a href="index.html">Main Page</a></li>
-              <li><a href="activities.html">Activities</a>
+              <li><a href="activities.html">Activities</a></li>
               <li><a href="resources.html">Resources</a></li>
-              <li><a href="aboutus.html">About us</a></li>
+              <li><a href="aboutUs.html">About us</a></li>
```

Comment:
Hey! this un-does my change that was just merged in: making all filenames lower-case. I think we should only use lower case because we're running into problems due to the different operating systems we're using. (Linux vs. OS X)